test/unit/dashboard/frontend-events.test.js

```diff
@@ -0,0 +1,146 @@
+const expect = require('chai').expect;
+const sinon = require('sinon');
+
+// unit to test
+const handlers = require('../../../dashboard/server/api/frontendApi');
```

Comment:
If we're testing `dashboard/server/api/frontendApi`, we should keep this test in `test/unit/dashboard/server/api/frontendApi.test.js`.

Reply:
Refactored every test file to similar names as in the code and made the same structure as well!